section4/exercises/classMethods.js

```diff
@@ -28,13 +28,37 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
-  }
-
-  // ADD CODE
+    this.toppings = toppings;
+    }
+    changeProtein() {
+      this.protein = "shrimp";
+    }
+    addTopping() {
```

Comment:
I might name these methods changeProteinToShrimp or addGuac. They have a limited range. Try utilizing parameters with these methods to respond to variable input.